gameShop/src/main/java/org/example/Main.java

```diff
@@ -0,0 +1,75 @@
+package org.example;
+
+import org.example.service.*;
+import org.example.service.menu.AuthorizationMenuService;
+import org.example.service.menu.ShopMenuService;
+import org.example.service.menu.UserMenuService;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Scanner;
+
+public class Main {
+
+    public static void main(String[] args) {
+        try(Scanner scanner = new Scanner(System.in)) {
+            scanner.useDelimiter("\n");
+            Singleton singleton = Singleton.getInstance(ConnectionSingleton.getConnection());
```

Comment:
Singleton is a representation of the class type, use it as a class name is bad idea